### ISSUE TITLE
Shellcheck

### ISSUE THREAD

**Issue by Cody-Learner** [clawoflight/aursec#27]
Not sure how it works, but did you transpose some digits somewhere other than the comments in the aursec-parse-srcinfo script? Does shellcheck read the commented line in the script?

ie: 
In bin/aursec-parse-srcinfo line 26:
	if [[ "$val" =~ .*::(.*git(@|\+http).*|.*svn(\+http|://).*|.*bzr\+http.*|.*hg\+http.*) ]]; then
                            ^-- SC1009: The mentioned parser error was in this regex grouping.
                                  ^-- SC1073: Couldn't parse this regex grouping.
                                      ^-- SC1072: Unexpected . Fix any mentioned problems and try again.
-vs-

#!/bin/bash
# shellcheck disable=SC1702,SC1703,SC1009
# Small script that parses a .SRCINFO file
# and outputs any vcs download folders that were found.

I tried rebuilding after today's update on the AUR and still can't.

**Comment by clawoflight** [comment]
Yeah, I don't understand why it doesn't work with the Arch package...
As you can see, it worked in CI, where shellcheck is built from git: https://circleci.com/gh/clawoflight/aursec/307

Also, welcome to github! :)

**Comment by clawoflight** [comment]
These are multiple upstream bugs:

The disabling doesn't work per [shellcheck/#1118](https://github.com/koalaman/shellcheck/issues/1118), and the wiki post on [SC1702](https://github.com/koalaman/shellcheck/wiki/SC1072) may be applicable as well.

Wait and see for the next upstream release, I guess...
and
